standards/views.py

```diff
@@ -53,6 +74,26 @@ def by_unit(request, slug, unit_slug):
     return render(request, 'standards/curriculum_nogrid.html', {'curriculum': curriculum, 'unit': unit})
 
 
+def by_unit_csv(request, slug, unit_slug):
+    curriculum = get_object_or_404(Curriculum, slug=slug)
+    unit = get_object_or_404(Unit, curriculum=curriculum, slug=unit_slug)
+
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment; filename="%s_%s_standards.csv"' % (curriculum.slug, unit.slug)
+
+    writer = csv.writer(response)
+    writer.writerow(['curriculum', 'unit', 'lesson #', 'lesson name',
+                     'standard framework', 'standard', 'cross curricular opportunity'])
+    for lesson in unit.lessons:
+        for standard in lesson.standards.all():
+            writer.writerow([curriculum.slug, unit.slug, 'lesson %d' % lesson.number, lesson.title,
+                             standard.framework.slug, standard.shortcode, ''])
+        for standard in lesson.opportunity_standards.all():
+            writer.writerow([curriculum.slug, unit.slug, 'lesson %d' % lesson.number, lesson.title,
+                             standard.framework.slug, standard.shortcode, 'X'])
+    return response
```

Comment:
it might make things simpler to give Lesson something like a `to_csv` method that can be used here and above

Reply:
This is a min bar to address the immediate need. I plan to come back here and flesh out a better solution later.